src/installation/system_requirements.md

```diff
@@ -18,3 +18,7 @@ For the Epinio server, and related deployments we recommend to consider the foll
 In addition, extensive requirements for your workload (apps) would add to that.
 
 A default storage class (with annotation `storageclass.kubernetes.io/is-default-class: "true"`) is needed.
+
+While Kubernetes 1.22 is supported there is an issue when the container runtime is `containerd > 1.5.6`.
+because the [pack cli](https://github.com/buildpacks/pack) is putting too much information inside the
```

Comment:
Redoing the suggestion then. Right now the phrasing is broken with an incomplete sentence due to the other requests applied.
```suggestion
The [pack cli](https://github.com/buildpacks/pack) is placing too much information into the
```